core/v2.go

```diff
@@ -225,28 +221,34 @@ func (core *Core) v2API() *route.Router {
 	r.Dispatch("GET /v2/auth/providers", func(r *route.Request) { // {{{
 		l := make([]v2AuthProvider, 0)
 
-		usage := r.Param("usage", "")
+		typ := r.Param("for", "cli")
 		for _, auth := range core.auth {
-			if usage == "" || auth.Usage == usage {
-				l = append(l, v2AuthProvider{
-					Name:       auth.Name,
-					Identifier: auth.Identifier,
-					Type:       auth.Backend,
-					Usage:      auth.Usage,
-				})
+			if auth.Backend == "token" && typ != "cli" {
```

Comment:
@thomasmmitchell this is where we hide the token backend from the list of available auth providers, unless you want CLI providers.